src/components/info-tile/info-tile.tsx

```diff
@@ -0,0 +1,273 @@
+            '. ' +
+            this.checkProps(this?.linkTitle);
+
+        const link = !this.disabled ? this.href : '#';
+
+        return [
+            <a
+                title={this.linkTitle}
+                href={link}
+                target={this.target}
+                tabindex="0"
+                aria-label={extendedAriaLabel}
+                aria-disabled={this.disabled}
+                class={{
+                    'is-clickable':
+                        (this.clickable || this.href) && !this.disabled,
+                    'has-circular-progress': !!this.progressValue,
+                }}
+            >
+                {this.renderIcon()}
+                {this.renderProgress()}
+                <div class="value-group">
+                    {this.renderPrefix()}
+                    <div class="value-suffix">
+                        {this.renderValue()}
+                        {this.renderSuffix()}
+                    </div>
+                    {this.renderSpinner()}
+                </div>
+                {this.renderLabel()}
+            </a>,
+            this.renderNotification(),
+        ];
+    }
+
+    private checkProps(propValue) {
+        return !propValue ? '' : propValue + ' ';
+    }
+
+    private renderPrefix = () => {
+        if (this.prefix) {
+            return <span class="prefix">{this.prefix}</span>;
+        }
+    };
+
+    private renderValue = () => {
+        const chCount = this.value.toString().length;
+
+        if (this.value) {
+            return (
+                <span
+                    class={{
+                        value: true,
+                        [`ch-${chCount}`]: true,
+                    }}
+                >
+                    {this.value}
+                </span>
+            );
+        }
+    };
+
+    private renderSuffix = () => {
+        if (this.suffix) {
+            return <span class="suffix">{this.suffix}</span>;
+        }
+    };
+
+    private renderIcon = () => {
+        if (this.icon) {
+            return <limel-icon class="icon" name={this.icon} />;
+        }
+    };
+
+    private renderProgress = () => {
+        if (this.progressValue) {
+            return (
+                <limel-circular-progress
+                    class="progress"
+                    prefix={this.progressPrefix}
+                    value={this.progressValue}
+                    suffix={this.progressSuffix}
+                    maxValue={this.progressMaxValue}
+                    displayPercentageColors={this.displayPercentageColors}
+                />
+            );
+        }
+    };
+
+    private renderLabel = () => {
+        if (this.label) {
+            return <span class="label">{this.label}</span>;
+        }
+    };
+
+    private renderNotification = () => {
+        if (this.badge) {
+            return <limel-badge label={this.badge} />;
+        }
+    };
+
+    private renderSpinner = () => {
+        if (this.loading) {
```

Comment:
I also think we should add `aria-live` or something similar in the `value` div or the whole component, when the value has been updated (?) or while it's being updated (?). This guides is so poorly written (for me at least), which I can't really figure out what to do or when to. do it. Need to do more research.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live

Reply:
Do you understand what they are saying in this guide @adrianschmidt ? 

I don't understand which one we should do: 
- while `loading` is `true`, we should also have `aria-live="polite"` on the `<a>`, and then remove it as soon as `loading` is `false` again
- or `aria-live="polite"` should always be there 🤔 (this sounds quite stupid)

Reply:
I'm not sure, but I think it needs to be there all the time, because if you remove it when removing the spinner, then you're telling the browser to no longer announce changes to that region, so the change you did while also removing the attribute probably won't get announced…

Reply:
But we'd not want to announce every change perhaps, at least not the very first when creating the component. Perhaps it should be added when loading is true, and then removed with a timer after loading becomes false. I think we really need to test it with a screenreader to  be sure…

Reply:
let's skip it for now. We add this as an issue for later.

Reply:
https://github.com/Lundalogik/lime-elements/issues/1973